code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -261,6 +261,8 @@ var/list/special_fruits = list()
 			spark(M) //Two set of sparks, one before the teleport and one after. //Sure then ?
 	return 1
 
+//Types blacklisted from appearing as products of strange seeds and no-fruit.
+var/list/strange_seed_product_blacklist = list(subtypesof(/obj/item/weapon/reagent_containers/food/snacks/grown/clover/)) //Otherwise the selection would be biased by the relatively large number of multiple leaf-number-specific subtypes - the base type with randomized leaves is still valid.
```

Comment:
I think this is a list with one element, which is a list; instead I think what you want is 
`= list() + subtypesof(/obj/item/weapon/reagent_containers/food/snacks/grown/clover)`
even then, you don't need the empty list; we'll expect whoever adds next to have the intelligence to add on without a blank list prompting them to (I appreciate the effort at clarity though)

Reply:
I wasn't totally sure if `subtypesof` returns a list in the case where there's only one subtype, rather than a naked typepath, so I wanted to wrap it in a list to be sure.. though since there's more than one here it shouldn't be an issue either way.